src/utils/packageUtils.ts

```diff
@@ -227,17 +226,18 @@ export function escapeInstallationKey(key?: string): Nullable<string> {
  */
 // eslint-disable-next-line @typescript-eslint/require-await
 export async function getContainerOptions(
-  packageIds: string[],
+  packageIds: string | undefined | Array<string | undefined>,
```

Comment:
This looks worse to me as an API.  Why do we have to do this?

Reply:
Will look